bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/DbLedgerStorageStats.java

```diff
@@ -85,16 +85,16 @@ class DbLedgerStorageStats {
     private final OpStatsLogger readEntryStats;
     @StatsDoc(
             name = READ_ENTRY_LOCATIONS_INDEX_TIME,
-            help = "time spent reading entries from the locations index of the db ledger storage engine",
+            help = "operation stats of reading entries from the locations index of the db ledger storage engine",
             parent = READ_ENTRY
     )
-    private final Counter readFromLocationIndexTime;
+    private final OpStatsLogger readFromLocationIndexTime;
     @StatsDoc(
             name = READ_ENTRYLOG_TIME,
-            help = "time spent reading entries from the entry log files of the db ledger storage engine",
+            help = "operation stats of reading entries from the entry log files of the db ledger storage engine",
             parent = READ_ENTRY
     )
-    private final Counter readFromEntryLogTime;
+    private final OpStatsLogger readFromEntryLogTime;
```

Comment:
Break change for metric readFromEntryLogTime,  I think this may need to send the mail vote. @eolivelli , please help evaluate it.